File: php/examples/post_order_items_requiring_attention.php
<?php
require_once 'base_request.php';

// Example of POST /order-items/requiring-attention

$debug = true;
$company_id = $_ENV['COMPANY_ID'];
$order_item_id = $_ENV['ORDER_ITEM_ID'];

$file_path = '../data/form_data_ein_tax_id.json';
$json_data = file_get_contents($file_path);
$form_data = json_decode($json_data);

$request_data = array(
    'company_id'    => $company_id,
    'order_item_id' => $order_item_id,
    'form_data'     => $form_data
);

$request_params = '';

$json_body = json_encode($request_data);

send_request('POST', '/order-items/requiring-attention', $request_params, $json_body);

File: csharp/examples/PostOrderItemsRequiringAttention.cs
using System;
using System.IO;
using Newtonsoft.Json.Linq;

namespace Examples.examples
{
    // Example of POST /order-items/requiring-attention
    
    public class PostOrderItemsRequiringAttention : BaseRequest
    {
        private static string _companyId;
        private static string _orderItemId;
        private static JObject _formData;

		public PostOrderItemsRequiringAttention()
		{
            _companyId = Environment.GetEnvironmentVariable("COMPANY_ID");
            _orderItemId = Environment.GetEnvironmentVariable("ORDER_ITEM_ID");
            loadFormData("../data/form_data.json");
        }

        public PostOrderItemsRequiringAttention(string companyId, string orderItemId)
        {
            _companyId = companyId;
            _orderItemId = orderItemId;
            loadFormData("../data/form_data_ein_tax_id.json");
        }

        public override string SendRequest()
        {
            var body = Newtonsoft.Json.JsonConvert.SerializeObject(new
            {
                company_id = _companyId,
                order_item_id = _orderItemId,
                form_data = _formData
            });
            Console.WriteLine($"PostOrderItemsRequiringAttention: {body}");
            return PostRequest("order-items/requiring-attention", body);
        }

        private void loadFormData(string formDataFilePath)
        {
            string currentDirectory = Environment.CurrentDirectory;
            string relativePath = formDataFilePath;
            string filePath = Path.Combine(currentDirectory, relativePath);
            string jsonData = File.ReadAllText(filePath);
            _formData = JObject.Parse(jsonData);
        }
    }
}